app/metrics/algolia.py

```diff
@@ -0,0 +1,15 @@
+from algoliasearch.search_client import SearchClient
+from app.config import Config
+
+APP_ID = Config.ALGOLIA_APP_ID
+API_KEY = Config.ALGOLIA_API_KEY
+
+
+def init_algolia_client():
+    return SearchClient.create(APP_ID, API_KEY)
+
+
+def get_dataset_count(client):
+    index = client.init_index("k-core_dev")
```

Comment:
"k-core_dev" should be an environment variable. For staging we can look at k-core_dev, but for prod we should be looking at SPARC_pr

Reply:
oups sorry! fixed now